packages/ripple-keypairs/src/index.ts

```diff
@@ -38,7 +38,7 @@ function generateSeed(
   const entropy = options.entropy
     ? options.entropy.slice(0, 16)
     : randomBytes(16)
-  const type = options.algorithm === 'ed25519' ? 'ed25519' : 'secp256k1'
+  const type = options.algorithm === 'ecdsa-secp256k1' ? 'secp256k1' : 'ed25519'
```

Comment:
Why is this behavior changed in this PR? That should be in a separate PR.

Reply:
thanks. I had pushed these changed by mistake

Reply:
fixed in [dfb14d3](https://github.com/XRPLF/xrpl.js/pull/2659/commits/dfb14d3eab2b9f1bd1dba9eb9259fed9f804f0b0)